ydb/docs/en/core/concepts/datamodel/view.md

```diff
@@ -0,0 +1,76 @@
+
+{% endnote %}
+
+## View invalidation
+
+If you drop a table that a view references, the view will become invalid. Queries against it will fail with an error caused by referencing a table that does not exist. To make the view valid again, you must provide a queryable entity with the same name (by creating or renaming a table or another view). It needs to have a schema compatible with the deleted one. The dependencies of views on tables and other views are not tracked. A `SELECT` from a view is executed like a `SELECT` from a subquery would, without any prior checks of validity. You would know that the view's query became invalid only at the moment of its execution. This approach will change in future releases: {{ ydb-short-name }} will start tracking the view's dependencies, and the default behavior would be to forbid dropping a table if there's a view referencing it.
+
+## Performance
+
+Queries are executed in two steps:
+1. compilation
+2. execution of the compiled code
+
+The resulting compiled code contains no evidence that the query was made using views because all the references to views should have been replaced during compilation by the queries that they represent. In practice, there must be no difference in the execution time of the compiled code (step 2) for queries made using views versus queries directly reading from the underlying tables.
+
+However, users might notice a little increase in the compilation time of the queries made using views compared to the compilation time of the same queries written directly. It happens because a statement reading from a view:
+```sql
+SELECT * FROM a_view;
+```
+is compiled similarly to a statement reading from a subquery:
+```sql
+SELECT * FROM (SELECT * FROM underlying_table);
+```
+but with an additional overhead of loading data from the schema object `a_view`.
+
+Please note that if you execute the same query over and over again, like:
+```sql
+-- execute multiple times
+SELECT * FROM hot_view;
+```
+compilation results will be cached on the {{ ydb-short-name }} server side, and you will not notice any difference in the performance of queries using views and direct queries.
+
+## View redefinition lag
+
+### Query compilation cache
+
+{{ ydb-short-name }} caches query compilation results on the server side for efficiency. For small queries like `SELECT 1;` compilation can take up to a hundred times more CPU time than the execution. The cache entry is searched by the text of the query and some additional information, such as a user SID.
+
+The cache is automatically updated by {{ ydb-short-name }} to stay on track with the changes made to the objects the query references. However, in the case of views, the cache is not updated in the same transaction in which the object's definition changes. It happens with a little delay.
+
+### Problem statement
+
+Imagine the following situation.
+
+Alice continuously executes the following query:
```

Comment:
```suggestion
### Problem example

Let's consider the following situation. Alice repeatedly executes the following query:
```

Reply:
It seems to me that "Problem example" is an uncommon phase in English. I would change that to "Example of the problem". The rest of the suggestion is perfect